modules/management/kubectl-apply/main.tf

```diff
@@ -48,3 +53,38 @@ module "kubectl" {
     http    = http.h
   }
 }
+
+module "install_kueue" {
+  source            = "./kubectl"
+  source_path       = local.install_kueue ? local.kueue_install_source : null
+  server_side_apply = true
+
+  providers = {
+    kubectl = kubectl
+    http    = http.h
+  }
+}
+
+module "install_jobset" {
+  source            = "./kubectl"
+  source_path       = local.install_jobset ? local.jobset_install_source : null
+  server_side_apply = true
+
+  providers = {
+    kubectl = kubectl
+    http    = http.h
+  }
+}
+
+module "configure_kueue" {
+  source      = "./kubectl"
+  source_path = try(var.kueue.config_path, "")
```

Comment:
I'd control if local.install_kueue is true, because it will apply configuration manifest if it's provided but `install` setting is false.

Reply:
Done